### PR TITLE
#17: Moved abstract scalar function test to virtual-schema-common-jdbc

### LINKED ISSUE #17: Move abstract scalar function test to virtual-schema-common-jdbc

### PR DESCRIPTION
**Pull request by jakobbraun**
Closes #17

### CODE REVIEW

File: doc/changes/changes_2.0.0.md [hunk 1]
```diff
@@ -6,8 +6,7 @@ Code name: Update supported Scalar Functions and remove `SQL_DIALECT` property
 
 The `SQL_DIALECT` property used when executing a `CREATE VIRTUAL SCHEMA` from the Exasol database is obsolete from this version. Please, do not provide this property anymore.
 
-Also, the list of supported Scalar Functions has been updated.
-The following Scalar Dunctions are not supported anymore:
+Also, the list of supported Scalar Functions has been updated. The following Scalar Dunctions are not supported anymore:
```
Comment (chiaradiamarcelo) [inline doc/changes/changes_2.0.0.md:9]:
```suggestion
Also, the list of supported Scalar Functions has been updated. The following Scalar Functions are not supported anymore:
```

File: doc/changes/changes_2.0.0.md [hunk 2]
```diff
@@ -29,6 +28,7 @@ The following new Scalar Functions are now supported:
 
 * #25: Refactored ScalarFunctionTests
 * #32: Removed `SQL_DIALECT` property
+* #17: Moved abstract scalar function test to virtual-schema-common-jdbc
```
Comment (chiaradiamarcelo) [inline doc/changes/changes_2.0.0.md:31]: can we maybe order this list by ticket number?

```
* #17: Moved abstract scalar function test to virtual-schema-common-jdbc
* #25: Refactored ScalarFunctionTests
* #32: Removed `SQL_DIALECT` property
```

File: pom.xml [hunk 3]
```diff
@@ -124,12 +124,6 @@
             <version>4.13.2</version>
             <scope>test</scope>
         </dependency>
-        <dependency>
-            <groupId>org.yaml</groupId>
-            <artifactId>snakeyaml</artifactId>
```
Comment (chiaradiamarcelo) [inline pom.xml:129]: We should add an entry of the removed dependency on the changes file

File: pom.xml [hunk 4]
```diff
@@ -148,6 +142,11 @@
             <version>0.1.0</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.exasol</groupId>
+            <artifactId>virtual-schema-shared-integration-tests</artifactId>
```
Comment (chiaradiamarcelo) [inline pom.xml:147]: same for this one

Comment (chiaradiamarcelo) [reply pom.xml:147]: we should probably add it in the `README` file as well

File: doc/changes/changes_2.0.0.md (hunk 2 above)
Comment (jakobbraun) [reply doc/changes/changes_2.0.0.md:31]: it's ordered chronologically.

**Review by jakobbraun — COMMENTED** [review]
I think we can ignore the dependencies file here as long as we switch to project-keeper 0.6.0 when releasing this repo.

**Review by chiaradiamarcelo — APPROVED** [review]

**Pull request merged**